data/gui/window/preferences/03_display.cfg

```diff
@@ -272,7 +272,7 @@
 			[toggle_button]
 				id = "fps_limiter"
 				label = _ "Limit FPS"
-				tooltip = _ "Disabling this increases CPU usage to 100% but may slightly improve performance at high resolutions"
+				tooltip = _ "Disabling this increases CPU usage, but may slightly improve performance (requires restart to take effect)"
```

Comment:
Thanks for the PR! I think the way this tooltip is worded may lead some players to think they need to reboot their computers for the change to take effect.

Reply:
Could be. I thought this wording was rather common in other games, too, though.
Do you have a suggestion how to improve it?

Reply:
Sorry for the late answer. How about `...may slightly improve performance. (Any change to this setting will only take effect after you restart Wesnoth.)`?